### ISSUE TITLE
I see `print` outputs twice in Debug Console when launch config uses `"console": "internalConsole"`

### ISSUE THREAD

**Issue by bersbersbers** [microsoft/debugpy#827]
Issue Type: <b>Bug</b>

Launch config:
```json
{
    "name": "Python: Current File",
    "type": "python",
    "request": "launch",
    "program": "${file}",
    "console": "internalConsole",
}
```

Then debug any file.

In Debug Console, enter `print(2)`.

# Expected Behaviour

I see "2" (once)

## Expected vs. Actual Behaviour

![image](https://user-images.githubusercontent.com/12128514/151137211-c302ddb8-4887-4ee1-869c-cda2176e76f8.png)

# Diagnostic data

-   Python version (& distribution if applicable, e.g. Anaconda): 3.10.0
-   Type of virtual environment used (e.g. conda, venv, virtualenv, etc.): Global
-   Value of the `python.languageServer` setting: Default

Extension version: 2021.12.1559732655
VS Code version: Code 1.63.2 (899d46d82c4c95423fb7e10e68eba52050e30ba3, 2021-12-15T09:40:02.816Z)
OS version: Windows_NT x64 10.0.19044
Restricted Mode: No
Remote OS version: Linux x64 5.3.18-lp152.44-default

**Comment by fabioz** [comment]
This was actually fixed in #678 (note that this still hasn't made it into a new release...)

**Comment by fabioz** [comment]
This happens because when evaluating we capture the output and send it to be shown in the `DEBUG CONSOLE`, but in this case the output was already going there, so, it's shown twice.

I'm taking a look on how to fix it.
